### ISSUE TITLE
counter locations

### ISSUE THREAD

**Issue by redpandah** [Cycling4Students/Group-Project#2]
<img width="557" alt="screen shot 2017-12-05 at 11 08 24" src="https://user-images.githubusercontent.com/33317778/33604359-948b2bc6-d9ad-11e7-93e9-920af7e5b6e4.png">

These are the coordinations of 48 counters! Shall we arrange the map accordingly and include the previous counter numbers of 48 as a footnote?

**Comment by sthuretta72** [comment]
amazing, you found them! yes, that's a good idea if we have time!

**Comment by sthuretta72** [comment]
yes, that's a good idea if we have time! we did work with the previous counters in mind but this is good as a natural development of the project i guess ~
